connect/settings.py

```diff
@@ -411,6 +412,8 @@
 INTELIGENCE_GRPC_ENDPOINT = env.str("INTELIGENCE_GRPC_ENDPOINT")
 INTEGRATIONS_GRPC_ENDPOINT = env.str("INTEGRATIONS_GRPC_ENDPOINT")
 
+INTELLIGENCE_REST_ENDPOINT = env.str("INTELLIGENCE_REST_ENDPOINT")
```

Comment:
Add a default value(can be a local value),  so it does not throw a env error (optional)